apps/authentication-service/src/resources/clients/clientsRoutes.ts

```diff
@@ -0,0 +1,31 @@
+import express, { Application, RequestHandler } from "express";
+
+import { validateAccessToken } from "@/middleware/authMiddleware";
+import {
+  clientsCreateV1,
+  clientsDeleteV1,
+  clientsGetV1,
+} from "@/resources/clients/clientsRoutesHandlers";
+
+const clientsRouter = express.Router();
+
+export const clientsRoutes = (app: Application): void => {
+  // Clients v1
+  app.use("/v1/clients", clientsRouter);
```

Comment:
This feels a little weird to me. I think it might make more sense to put this line in the `apps/authentication-service/src/init.ts` instead of `clientsRoutes(app);` and then in this file export the clientsRouter. What do you think of that idea?

Reply:
When/If we do create a new version of the routes, we'll need a new router to handle those. I'd rather the routers lived in the clientRoutes file, to keep the client route logic together, instead of the `init.ts` file. 

I could be way off on this, as this is my first api, so please feel free to correct me if my logic is flawed.

Reply:
I think in the event that we do have to make a new version these files could get really big and messy. Especially the clientsRoutesHandlers file could get really big because the file isn't versioned, instead the methods are versioned. So if we have 3 or 4 versions that file would contain all the same methods for all the versions.

I think you could potentially keep this file as it is but version the `clientsRoutesHandlers` to `clientsRoutesHandlersV1` and then in this file if you have multiple versions you can import the methods from each version and it keeps the routes handler files much smaller and separated which I think would help code readability in the long run. If we get another version we would create clientsRoutesHandlersV2 and update this file to use those handlers.

I would also rename the `clientsRouter` variable in this file to `clientsRouterV1`, because if you have new versions you are going to have to define another clientsRouter in this file for each version and then assign the requests.

Reply:
Great feedback. Thanks @Craiting